### DISCUSSION TITLE
Icon documentation seems inaccurate

### DISCUSSION THREAD

**Discussion by R0N1n-dev** [quasarframework/quasar#16138] [General - Components / Directives / etc]
### Description

Instructions for webfont icons not working for lineawesome
![Screenshot from 2023-08-02 11-04-14](https://github.com/quasarframework/quasar/assets/83344872/1e47327c-f15e-4d02-b907-5130a3e67fed)
![Screenshot from 2023-08-02 11-12-54](https://github.com/quasarframework/quasar/assets/83344872/35820164-ffb5-4ade-ae6b-f183964efd5c)

### Documentation Section URL

https://quasar.dev/vue-components/icon

### Flavour

Vite Plugin (@quasar/vite-plugin)

### Areas

Quasar Extras (@quasar/extras)

**Comment by pdanpdan** [comment]
What instructions? What are you using? What ...?

**Reply by R0N1n-dev** [reply]
Instructions on the Icon page.

```html
<q-icon name='las la-atom' />
```

**Comment by R0N1n-dev** [comment]
![Screenshot from 2023-08-02 11-23-15](https://github.com/quasarframework/quasar/assets/83344872/cf12d75d-802b-40ea-8a36-7c3a56c79bb1)
![Screenshot from 2023-08-02 11-23-00](https://github.com/quasarframework/quasar/assets/83344872/30d39d1f-9334-4025-8bb6-e4bb5aad074a)
![Screenshot from 2023-08-02 11-04-14](https://github.com/quasarframework/quasar/assets/83344872/d58d28e0-bd2f-4d9b-9cf2-775b7b467451)

But as you can see, no icons appear

![Screenshot from 2023-08-02 11-24-52](https://github.com/quasarframework/quasar/assets/83344872/60388b76-5bd4-4bf3-8716-2ec81d5f6f77)

**Comment by R0N1n-dev** [comment]
Apparently the CSS file(main.css) that comes with the vite vue 3 template was messing everything up. On removing it, the icons show.
